_data/simple-icons.json

```diff
@@ -4203,7 +4203,7 @@
         {
             "title": "Deno",
             "hex": "000000",
```

Comment:
Have we ever considered using `#70FFAF` as the hex here? They seem to use it as their accent/brand color in some places, though not necessarily in the logo itself.

![CleanShot 2024-10-10 at 21 23 33](https://github.com/user-attachments/assets/21b7adec-1820-4579-a4b8-7da3e1c6eac0)

![image](https://github.com/user-attachments/assets/d1609f24-0c79-46ee-85cb-1ec814eab60e)

![image](https://github.com/user-attachments/assets/152eb5d8-453e-408a-bfa7-c354da6d5a2b)

Reply:
We should not use that color on the icon. They didn't use this color on the logo.

Reply:
What do you mean the homepage theme color is subject to change... everything is subject to change? We are changing the "blank-and-white Deno" logo literally right now 😅

Reply:
And for the record [our guidelines say](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#color-guidelines) that the brand color isn't necessarily the color of the icon, just the accent color the brand uses in general.

Reply:
They didn't use color on the logo. This is a color that has nothing to do with the icon, so we shouldn’t use it on the icon.

Reply:
Like I said, our guidelines don't say that the brand color must be from the icon. It looks pretty clear to me that Deno uses this green color as their "accent", so I don't see why not. We can always ask someone working at the company for confirmation too...

Reply:
It's official is just black-and-white. They even added a white background to the icon to ensure that the icon is always black-and-white.